src/nvim/options.lua

```diff
@@ -834,7 +836,7 @@ return {
       abbreviation = 'bh',
       cb = 'did_set_bufhidden',
       defaults = { if_true = '' },
-      values = { 'hide', 'unload', 'delete', 'wipe' },
+      values = { '', 'hide', 'unload', 'delete', 'wipe' },
```

Comment:
Nit: Is there a reason `''` is at the start here and at the end for other options?

Reply:
In earlier patches, having empty at the start caused some failures so I added it back after I made changes to C. Empty is also the default value for some of these options.